bigframes/session/__init__.py

```diff
@@ -792,6 +838,14 @@ def _read_ibis(
     def read_gbq_model(self, model_name: str):
         """Loads a BigQuery ML model from BigQuery.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> model_name = "bigframes-dev.bqml_tutorial.penguins_model"
```

Comment:
Likewise, a header for the actual sample:

```suggestion
            >>> bpd.options.display.progress_bar = None

        Read an existing BigQuery ML model.

            >>> model_name = "bigframes-dev.bqml_tutorial.penguins_model"
```